src/lxml/tests/test_dtd.py

```diff
@@ -24,6 +24,20 @@ def test_dtd_file(self):
 
         dtd = etree.DTD(fileInTestDir("test.dtd"))
         self.assertTrue(dtd.validate(root))
+    
+    def test_dtd_file_pathlike(self):
+        parse = etree.parse
+        tree = parse(fileInTestDir("test.xml"))
+        root = tree.getroot()
+
+        class Path(object):
+            def __init__(self,path):
+                self.path = path
+            def __fspath__(self):
+                return self.path
```

Comment:
There is a `common_imports.py` helper module for tests. You can just move this class there, so that you don't have to repeat it in all test methods. Maybe name it `SimpleFSPath` then, or so, to distinguish it from `pathlib`.

Reply:
Done.